src/ILLink.RoslynAnalyzer/RequiresAnalyzerBase.cs

```diff
@@ -360,6 +367,11 @@ protected bool IsMemberInRequiresScope (ISymbol containingSymbol)
 		/// True if the target of a call is considered to be annotated with the Requires... attribute
 		/// </summary>
 		protected bool TargetHasRequiresAttribute (ISymbol member, [NotNullWhen (returnValue: true)] out AttributeData? requiresAttribute)
```

Comment:
I'd change the name here to indicate this one is for overriding. Maybe something like `TargetHasRequiresAttributeForOverriding`

Reply:
Doesn't the `protected` vs `private` already indicate which should be overridden?

Reply:
Sorry, I didn't mean which method should be overridden, I meant that this method should be used for determining mismatches when overriding. But given Sven's feedback that this is done differently in the linker, maybe the right answer is to go back to the old API and make it behave as this method does.